docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.9'
+services:
+  nginx:
+    build: ./nginx
+    ports:
+      - 8008:80
+    depends_on:
+      - web
+  web:
+    build: 
+      context: .
+    command: gunicorn game_nerd.wsgi:application --bind 0.0.0.0:8000
+    volumes:
+      - ./game_nerd/:/Users/noname/PycharmProjects/django/game_ners
```

Comment:
you have a typo - `ners`

same here with local path, probably should be `- ./game_nerd:/game_nerd